libraries/AP_Logger/LogFile.cpp

```diff
@@ -244,6 +244,41 @@ void AP_Logger::Write_RCOUT(void)
         chan14        : hal.rcout->read(13)
     };
     WriteBlock(&pkt, sizeof(pkt));
+
+#if NUM_SERVO_CHANNELS >= 15
+    const struct log_RCOUT2 pkt2{
+        LOG_PACKET_HEADER_INIT(LOG_RCOUT2_MSG),
```

Comment:
This is incorrectly using the same struct name. Perhaps we should call it log_RCOUT15_18?